front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -84,6 +81,17 @@ const usedModelConfigs = [
   GEMINI_PRO_DEFAULT_MODEL_CONFIG,
 ];
 
+// Avatar URLs
+const BASE_URL = "https://dust.tt/";
+const buildAvatarUrl = (basePath: string, fileName: string) =>
```

Comment:
You can use `path` here to format 😛.

Reply:
hmm sorry copilot refused :sweat_smile: 
![image](https://github.com/dust-tt/dust/assets/5437393/11c881db-a2dd-4df2-986f-55e1eb43da10)

Reply:
Oh, I meant to say 'URL' in my previous message. No worries, though, it's all good!